dashdotdb/__init__.py

```diff
@@ -6,16 +6,16 @@
 from connexion.resolver import RestyResolver
 
 from dashdotdb.models.base import db
-from dashdotdb.models import imagemanifestvuln  # type: ignore  # noqa: F401
+from dashdotdb.models import dashdotdb  # type: ignore  # noqa: F401
 
 
 DATABASE_URL = os.environ.get('DASHDOTDB_DATABASE_URL')
 if DATABASE_URL is None:
-    DATABASE_HOST = os.environ['DATABASE_HOST']
-    DATABASE_PORT = os.environ['DATABASE_PORT']
-    DATABASE_USERNAME = os.environ['DATABASE_USERNAME']
-    DATABASE_PASSWORD = os.environ['DATABASE_PASSWORD']
-    DATABASE_NAME = os.environ['DATABASE_NAME']
+    DATABASE_HOST = os.getenv('DATABASE_HOST') or 'localhost'
+    DATABASE_PORT = os.getenv('DATABASE_PORT') or '5432'
+    DATABASE_USERNAME = os.getenv('DATABASE_USERNAME') or 'postgres'
+    DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD') or 'postgres'
+    DATABASE_NAME = os.getenv('DATABASE_NAME') or 'postgres'
```

Comment:
you have the DASHDOTDB_DATABASE_URL working now and you're exporting it in your `Makefile.devhelpers`. I'd rather have the missing env-var error in production (not having defaults).

Reply:
that is a good point re: erroring if missing..  i'll remove that and push up.

Reply:
reverted back to os.environ[]